apps/web/src/components/Sidebar/DocumentTree/index.tsx

```diff
@@ -20,13 +22,14 @@ export function CreateNode({ parentId }: { parentId?: number }) {
 }
 
 function CreateFolder({ parentId }: { parentId?: number }) {
-  const { create } = useDocumentVersions({ staged: true })
+  const { execute } = useServerAction(createDocumentVersionAction)
   return (
     <button
-      onClick={() =>
-        create({
+      onClick={async () =>
+        await execute({
+          commitUuid: 'HEAD',
           parentId,
-          documentType: 'folder' as DocumentType.Folder,
+          documentType: DocumentType.Folder,
```

Comment:
this fails because DocumentType is declared in a file that has other non-client deps like drizzle. You'll have to move the declaration of documenttype if u wanna make it work